server/db/database.js

```diff
@@ -1,13 +0,0 @@
-const { Client } = require('pg');
```

Comment:
@Mantaco was this file deleted? Maybe I'm missing something, but this shouldn't be deleted right - at least my code still depends on it

Reply:
It's deleted from the push, i added it to git ignore but that didnt work for some reason. I rely on a bit of a modification because my local computer is weird

Reply:
Weird - i think this might delete the file for everyone though, could be wrong